src/Core/src/ImageSources/iOS/ImageAnimationHelper.cs

```diff
@@ -0,0 +1,216 @@
+// Contains code from luberda-molinet/FFImageLoading
+// https://github.com/luberda-molinet/FFImageLoading/blob/bb675c6011b39ddccecbe6125d1853de81e6396a/source/FFImageLoading.Shared.IosMac/Decoders/GifDecoder.cs
```

Comment:
Should we add a note here?
https://github.com/dotnet/maui/blob/main/LICENSE.txt

Reply:
Added the license to the third party notices